src/admin-views/aggregator/settings.php

```diff
@@ -357,7 +357,18 @@
 if ( Tribe__Events__Aggregator::instance()->is_service_active() ) {
 	ob_start();
 	?>
-	<p><?php esc_html_e( 'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options.', 'the-events-calendar' );?></p>
+	<p>
+		<?php
+		printf(
+			esc_html__(
+				'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options. Check your Event Aggregator Service Status on the %1$sHelp page%2$s.',
+				'the-events-calendar'
+			),
+			'<a href="' . Tribe__Settings::instance()->get_url( array( 'page' => 'tribe-help' ) ) . '#tribe-tribe-aggregator-status">',
```

Comment:
Could we try to avoid links like that on the Domains, Suggestion:
```php
<?php
printf(
	esc_html__( 'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options. Check your Event Aggregator Service Status on the %s.', 'the-events-calendar' ),
	'<a href="' . Tribe__Settings::instance()->get_url( array( 'page' => 'tribe-help' ) ) . '#tribe-tribe-aggregator-status">' . esc_html__( 'Help page', 'the-events-calendar' ) . '</a>'
);
?>
```